community/slang/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+./configure
+make
+make DESTDIR="$1" install
```

Comment:
Looks like you'll have to specify `make -j1; make -j1 DESTDIR="$1" install` otherwise the build eats itself because it doesn't like parallel jobs...